knexfile.js

```diff
@@ -1,4 +1,4 @@
-export default {
+module.exports = {
```

Comment:
When we get some time we can figure this out. but yeah just use what's working for now.